### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by chenlujjj**

### CODE REVIEW

**Comment by chenlujjj** [comment]
> @chenlujjj can you sign the CLA? Then we can review.

Just signed

**Comment by svrnm** [comment]
@open-telemetry/collector-approvers PTAL

File: content/en/docs/collector/getting-started.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ the following:
 
 ```yaml
 otel-collector:
-  image: otel/opentelemetry-collector-contrib
+  image: otel/opentelemetry-collector
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/collector/getting-started.md:69]: @chenlujjj without the change you proposed is the docker compose working using the contrib collector?

**Comment by svrnm** [comment]
Closing this as we want to keep the `-contrib`.

**Pull request closed without merging**